src/chapters/introduction-and-setup/why-csharp.rst

```diff
@@ -0,0 +1,25 @@
+Why Learn C#?
+=============
+
+.. index:: ! C#, csharp
+
+This course is an introduction to **C#** and the .NET framework. C# is a
```

Comment:
this is funny, Jim tells me not to bolden anything unless its getting defined so when i wrote this bit for the java book, he had me remove the entry since we dont define the term until later. I've been trying to follow that rule since